docs/pages/starter-kit/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Starter Kit'
+---
+
+# Starter kit
+
+The starter kit is an example of an application running on the Liteflow infrastructure to showcase the uses of the infrastructure and/or be used as a starter kit to launch a product fast.
+
+This application is fork-able on this repository: https://github.com/liteflow-labs/marketplace-template.
+
+## Features included
+
+The Marketplace template includes the following features:
+
+- Open collections (User-generated content)
+  - Mint ERC-721
+  - Mint ERC-1155
+  - Creator's royalties support
+- Featured NFTs on the home page
+- Search system
+- Explore NFTs with filtering and sorting systems
+- Native currency support
+- List NFTs on sale
+  - Fixed price listing
+  - Timed auction listing
+- Make offers on NFTs
+  - Open offer
+  - Timed auction offer
```

Comment:
```suggestion
- Bid on NFTs
  - Open bid
  - Timed auction bid
```

Reply:
not sure about this one, need to check with @EmmanuelDrouin but we were pushing on offers rather than bid to be more accurate with what's the market uses

Reply:
@antho1404 keep offers, we are indeed trying to fix a naming issue we have for too long ;)